docs/E-Land_Requirements.rst

```diff
@@ -163,20 +167,19 @@ FUN-MVS-07 - Optimisation goal
 
 :Priority:  HIGH
 
-:Progress: In progress
+:Progress: Done
 
 :Progress message:
 
-In general, the MVS aims to minimize the energy supply cost of the local energy system. Additionally, following constraint can be activated:
+In general, the MVS aims to minimize the energy supply cost of the local energy system. Additionally, a number of :ref:`constraints <constraints-label>` can be activated:
 
-* Minimal renewable share constraint (see `here <https://mvs-eland.readthedocs.io/en/latest/Model_Assumptions.html#minimal-renewable-share-constraint>`__)
+* :ref:`Minimal renewable share constraint <constraint_minimal_renewable_factor>`
+* :ref:`Minimal degree of autonomy <constraint_minimal_degree_of_autonomy>`
+* :ref:`Maximum GHG emission constraint <constraint_maximum_emissions>`
+* :ref:`Net zero energy constraint <>`
```

Comment:
link is missing

Reply:
e98697e0521a0e86a1d7f6431cf9e6a088b5109a